### ISSUE TITLE
Automatic ReCaptcha solving

### ISSUE THREAD

**Issue by AVHon** [alexanderankin/ohio-covid-toolkit#1]
I figured I'd create an issue for discussing client-side technology for defeating ReCaptcha, which might work as an alternative to farming the work out to low-wage humans.

**Comment by AVHon** [comment]
My submission is @dessant's [Buster](https://github.com/dessant/buster), a browser extension that solves ReCaptcha by applying voice recognition to the audio challenge and simulating user input.

**Comment by dewineresign** [comment]
@AVHon good idea, do you think that can be run with puppeteer if we disabled headless mode?

**Comment by AVHon** [comment]
@dewineresign I would expect so, but I am definitely not a browser automation expert.
